Scripts/install_pst.sh

```diff
@@ -26,6 +26,7 @@ if pkg_installed sddm; then
 
```

Comment:
.

Reply:
The entry is there
```
 1) sddmtheme="Candy" ;;
        2) sddmtheme="Chilly" ;;
        *) sddmtheme="Corners" ;;
```

Reply:
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/b31c7011-5057-489a-a380-00d93b050994)

Reply:
No it won't shoe you should echo it first

Reply:
The selection is there but it will not be displayed

Try to run ` ./install_pst.sh ` to test 

to trigger this part of the script remove the t2  backup file

Reply:
where is the backup localted at

Reply:
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/47e98df9-0a4e-4de5-a641-f36d40447808)

Reply:
fixed 
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/0b56a83a-e193-4e1e-95a2-2fb2284815cd)